101-data-factory-blob-2-sql/azuredeploy.json

```diff
@@ -0,0 +1,194 @@
+{
+  "contentVersion": "1.0.0.0",
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "parameters": {
+    "dataFactoryName": {
+      "type": "string",
+      "metadata": {
+        "description": "Data Factory Account name"
+      }
+    },
+    "storageAccountName": {
```

Comment:
This will cause user input failures due to strict storage account requirements. I recommend 
1. taking in a prefix that's limited to 11 chars
2. toLower the input
3. concat the toLowered input to the output of a uniqueString for a unique storage account name